src/config.ts

```diff
@@ -254,8 +254,10 @@ const DEFAULTS = o({
     browser: "firefox",
     nativeinstallcmd:
         "curl -fsSl https://raw.githubusercontent.com/cmcaine/tridactyl/master/native/install.sh | bash",
-    win_nativeinstallcmd:
+    win_powershell_nativeinstallcmd:
         "Set-ExecutionPolicy Bypass -Scope Process -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/gsbabil/tridactyl/master/native/win_install.ps1'))",
+    win_cmdexe_nativeinstallcmd:
+        '@"%SystemRoot%\\System32\\WindowsPowerShell\\v1.0\\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString(\'https://raw.githubusercontent.com/gsbabil/tridactyl/master/native/win_install.ps1\'))"',
```

Comment:
I don't think ExecutionPolicy is needed here, either.

Reply:
Fixed [here][0].

[0]: https://github.com/gsbabil/tridactyl/commit/1e572c5ba5726f2789457bf15c695ff973a5d24d